src/components/pill/components/button-pill/config/index.js

```diff
@@ -0,0 +1,24 @@
+import { PILL_TYPE } from '../../../enums';
```

Comment:
Mag gerust gewoon `TYPE` heten, maar is misschien personal preference.

Reply:
Fixed.